site/_filters/md.js

```diff
@@ -0,0 +1,113 @@
+const md = require('markdown-it')();
```

Comment:
This file should be renamed something like "reference-support.js". It renders inline MD but also turns `{@link ...}` into useful things.